crates/autopilot/src/database/onchain_order_events.rs

```diff
@@ -133,8 +133,15 @@ impl<T: Sync + Send + Clone, W: Sync + Send + Clone> EventStoring<ContractEvent>
         events: Vec<EthContractEvent<ContractEvent>>,
         range: RangeInclusive<u64>,
     ) -> Result<()> {
-        let (custom_onchain_data, quotes, broadcasted_order_data, orders) =
-            self.extract_custom_and_general_order_data(events).await?;
+        let order_placement_events = events
+            .into_iter()
+            .filter(|EthContractEvent { data, .. }| {
```

Comment:
This is only temporary. THe next PR will also filter for the new cancellation event